resources/extruders/nwa3d_a5_extruder_0.def.json

```
@@ -0,0 +1,16 @@
+{
+    "id": "nwa3d_a5_extruder_0",
+    "version": 1,
```

It has to be `"version": 2`
```suggestion
    "version": 2,
```